content/cloud/variables.md

```diff
@@ -101,22 +101,29 @@ The resulting application manifest should look similar to the following:
 <!-- @nocpy -->
 
 ```toml
-spin_manifest_version = "1"
-description = "A Spin app with a dynamically updatable variable"
+spin_manifest_version = 2
+
+[application]
 name = "pw_checker"
-trigger = { type = "http", base = "/" }
 version = "0.1.0"
+authors = ["author <author@fermyon.com>"]
+description = "A Spin app with a dynamically updatable variable"
+
+[[trigger.http]]
+route = "/..."
+component = "pw-checker"
+
+[component.pw-checker]
+source = "target/wasm32-wasi/release/pw_checker.wasm"
+allowed_http_hosts = []
 
 [variables]
 password = { required = true }
 
-[[component]]
-id = "pw-checker"
-source = "app.wasm"
-[component.trigger]
-route = "/..."
-[component.build]
-command = "spin py2wasm app -o app.wasm"
+[component.pw-checker.build]
```

Comment:
I am a bit surprised to see the `[variables]` section between two parts of `component.pw-checker`.  Merge issue?